src/explorer/explorerTreeController.ts

```diff
@@ -38,7 +38,7 @@ implements vscode.TreeDataProvider<vscode.TreeItem> {
     this._connectionController.removeEventListener(
       DataServiceEventTypes.CONNECTIONS_DID_CHANGE,
       () => {
-        this.refresh();
+        void this.refresh();
```

Comment:
I guess this is as above right void not necessary? But in this case this was also ok here: 

```
async () => { await this.refresh() }
```

As far as i remember EventEmitter."method..." accepts any function and won't wait for the promise

Reply:
This code is outdated. The `this.refresh()` is sync now.

Reply:
This code is outdated. The `this.refresh()` is sync now.

Reply:
And in general wouldn't it be more descriptive to mark such function called as `void` to kind of let people who reads code know that emitter event does not wait for a response.